conans/client/tools/system_pm.py

```diff
@@ -86,11 +87,12 @@ def update(self):
         self._is_up_to_date = True
         self._tool.update()
 
-    def install(self, packages, update=True, force=False):
+    def install(self, packages, update=True, force=False, platforms=None):
```

Comment:
Won't be clearer to name it `arch_suffixes` or something like that instead of `platforms`?

Reply:
platforms is because MSBuild, which has a parameter called platforms. So I would prefer archs instead, because we could use suffix or prefix, depending the platform.

Reply:
renamed to "archs_names"